samples/net/sockets/http_server/src/static_web_resources/main.js

```diff
@@ -0,0 +1,45 @@
+async function fetchUptime()
```

Comment:
Should we add license headers in a javascript file? I couldn't find any existing `.js` files in the tree (understandably!)

Reply:
Never mind, CI answered the question for me